linkerd/tls/src/test/scala/io/l5d/clientTls/boundPathTest.scala

```diff
@@ -7,10 +7,21 @@ import io.buoyant.linkerd.TlsClientInitializer
 import org.scalatest.FunSuite
 
 class boundPathTest extends FunSuite {
-  test("sanity") {
+  test("static sanity") {
     static("hello", None).tlsClientPrep
   }
```

Comment:
Hmm, I don't think this test even belongs in this file. It appears to be defined in two places:

```
$ grep -r 'static("hello", None).tlsClientPrep' ./*
./linkerd/tls/src/test/scala/io/l5d/clientTls/boundPathTest.scala:    static("hello", None).tlsClientPrep
./linkerd/tls/src/test/scala/io/l5d/clientTls/staticTest.scala:    static("hello", None).tlsClientPrep
```

Can you just remove it from here?